.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test.md
```

Comment:
#- [ ] Good job adding the `test.md` file in the `.gitignore`. However, to exactly perform this change, there are some things you improve again. When a file is ignored by git ii should not appear on **Github** repo but in your case your `test.md` file is appeared on Github repo.
To fix that you should need: 
  1. Delete the `test.md` file from Github repo
  2. Then pull your changes locally
  3. then when you push again on github, you should make sure the `test.md` file doesn;t appear on Github repo